### DISCUSSION TITLE
UnityGoogleDrive.Error: HTTP/1.1 403 Forbidden

### DISCUSSION THREAD

**Discussion by 0060develop** [Elringus/UnityGoogleDrive#134] [Q&A]
hello! I found the following error while using your 'Unity Google Drive'. I have already received the API key. But I don't know where to put this. I couldn't find anything in the guide document. 

UnityGoogleDrive.Error: HTTP/1.1 403 Forbidden
Google Drive API Error Description: Code '403' Message: 'The request is missing a valid API key.'

Could you help me solve the problem?

**Comment by PtrKostenko** [comment]
App that was using uploading to google drive was working well until 19 of February
When I trying to test it from editor it throws same error as above

UnityGoogleDrive: HTTP/1.1 403 Forbidden
Google Drive API Error Description: Code '403' Message: 'The request is missing a valid API key.'
 - Domain: 'global' Reason: 'forbidden' Message: 'The request is missing a valid API key.' LocationType: '' Location: ''
UnityEngine.Debug:LogError(Object)
UnityGoogleDrive.GoogleDriveRequest`1:HandleWebRequestDone(AsyncOperation) (at Library/PackageCache/com.elringus.unitygoogledrive@fe300dd537/Runtime/GoogleDriveRequest.cs:241)
UnityEngine.AsyncOperation:InvokeCompletionEvent()
